run/logging-manual/test/system.test.js

```diff
@@ -0,0 +1,114 @@
+const assert = require('assert');
+const {Logging} = require('@google-cloud/logging');
+const request = require('got');
+
+const promisify = require('util').promisify;
+const setTimeoutPromise = promisify(setTimeout);
+
+// Support concurrency by setting the service name to something unique.
+// The service name must be the same used to deploy to Cloud Run.
+const service_name = process.env.CLOUD_RUN_SERVICE_NAME || require('../package').name;
+
+const logging = new Logging({
+  projectId: process.env.GOOGLE_CLOUD_PROJECT
+});
+
+const getLogEntriesPolling = async (filter, max_attempts) => {
+  const WRITE_CONSISTENCY_DELAY_MS = 10000;
+  const MAX_ATTEMPTS = max_attempts || 8;
+  let entries;
+
+  for (i = 0; i < MAX_ATTEMPTS; i++) {
+    await setTimeoutPromise(WRITE_CONSISTENCY_DELAY_MS);
```

Comment:
Can we use a dedicated retry library here? Maybe [`promise-retry`](https://github.com/IndigoUnited/node-promise-retry) perhaps?

Reply:
If necessary. I don't see a library saving many lines of code for the single time I need to do retries.

Reply:
On the other hand, if we wanted to turn this into a logging test helper, whatever makes it generally usable :)

Reply:
I feel like Node, idiomatically-speaking, favors the use of libraries. @fhinkel thoughts?

Reply:
I agree that Node tends to favor libraries, but still the question is:

* Is maintaining the updated dependency or maintaining the lines of code more burden?
* Is this use of the library going to drive meaningful community collaboration?
* Is adding the dependency going to appreciably affect build or deploy times? (`promise-retry` seems quite light. `moment` and `gcp-metadata` less so.)

Reply:
Personally, I prefer fewer libraries for samples. They can be black boxes unless they're well established libraries.